lang/en.json

```diff
@@ -262,10 +261,19 @@
+    "iveCheckedThisLpaAndImHappyToShowToCertificateProvider": "I’ve checked this LPA and I’m happy to show it to my certificate provider, {{ .CertificateProviderFullName }}",
+    "iveCheckedThisLpaAndImHappyToShareWithCertificateProvider": "I’ve checked this LPA and I’m happy for OPG to share it with my certificate provider, {{ .CertificateProviderFullName }}",
+    "onceYouClickCertificateProviderWillBeSentText": "Once you select the confirm button, your certificate provider will be sent a text telling them you have changed your LPA.",
     "whatHappensIfIChange": "What happens if I need to make changes later?",
-    "whatHappensIfIChangeDetails": "<p class=\"govuk-body\">If you want to change your LPA after you’ve had the conversation with your certificate provider, you can.</p> <p class=\"govuk-body\">After you’ve made the changes, you’ll need to have the conversation with your certificate provider again, so you can talk about the updated LPA.</p>",
-    "onceYouClickConfirmWarning": "Once you select the confirm button, you will not be able to make any more changes.",
+    "whatHappensIfIChangeDetails": "<p class=\"govuk-body\">You can still change your LPA after you’ve spoken to your certificate provider, if you have not signed your LPA yet.</p> <p class=\"govuk-body\">After you’ve made any changes, your certificate provider will be notified by text. You’ll then need to have another conversation with your certificate provider, to talk about your updated LPA.</p>",
+    "lpaDetailsSaved": "LPA details saved",
+    "lpaDetailsSavedContentOnFirstSubmissionCPCarryingOutRolePaper": "You should show your LPA to your certificate provider, {{ .CertificateProviderFullName }}.",
+    "lpaDetailsSavedContentOnFirstSubmissionCPCarryingOutRoleOnline": "We’ve sent an email to your certificate provider, {{ .CertificateProviderFullName }}, to tell them what they need to do next. You should tell them to expect an email from us.",
+    "lpaDetailsSavedContentOnSubsequentSubmissionCPCarryingOutRolePaper": "We’ve saved your changes and sent a text to your certificate provider, {{ .CertificateProviderFullName }}, to tell them that your LPA is ready for review. You should show them your LPA.",
+    "lpaDetailsSavedContentOnSubsequentSubmissionCPCarryingOutRoleOnline": "We’ve saved your changes and sent a text to your certificate provider, {{ .CertificateProviderFullName }}, to tell them that they should review your LPA online.",
+    "weveSentATextToCPContent": "<p class=\"govuk-body\">We’ve sent a text {{ .CertificateProviderFirstName }} to tell them you’ve made an LPA. We’ve also given them a link to some online guidance about being a certificate provider.</p> <p class=\"govuk-body\">If {{ .CertificateProviderFirstName }} needs help viewing this online, you should show them the guidance for certificate providers on GOV.UK (opens in a new tab).</p>",
```

Comment:
I'm guessing this should be "We’ve sent a text [to] ..."

Reply:
That definitely makes more sense - this was a copy and paste from figma but will update as its most likely a mistake in there.